formats/keep/add-keep-frontmatter.ts

```diff
@@ -0,0 +1,37 @@
+import { FileManager, TFile } from "obsidian";
+import { addAliasToFrontmatter, addTagToFrontmatter, sanitizeHashtag } from '../../util';
+import { KeepJson } from "./models/KeepJson";
+
+
+
+export async function addKeepFrontMatter(fileRef: TFile, keepJson: KeepJson, fileManager: FileManager) {
```

Comment:
This is quite inefficient because it keeps re-reading the file from disk, writing it back. Can we wrap it all in a single `processFrontMatter` call, and just pass the frontmatter object to each helper?

Reply:
Will do :)